src/Calendar/Navigation.jsx

```diff
@@ -136,7 +136,7 @@ export default class Navigation extends Component {
           style={{ flexGrow: 1 }}
           type="button"
         >
-          {label}
+          {this.props.navigationLabel(label)}
```

Comment:
navigationLabel is marked as not required. And that's actually good (see my other comment). For unit tests to pass, you should write something like that:

```js
  {
    this.props.navigationLabel ?
      this.props.navigationLabel(label) :
      label
  }
```

and this should do the trick.

Another thing would be to make it streamlined with tileContent. `tileContent` provides a single object argument of `({ date, view })`. So, I'd suggest in the same here.

```js
```js 
  const { activeStartDate: date, view } = this.props;

  (...)

  {
    this.props.navigationLabel ?
      this.props.navigationLabel({ date, view }):
      label
  }
```
```